### PR TITLE
Add func

### PR DESCRIPTION
**Pull request by jongio**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: scripts/prepdocs.ps1 [hunk 1]
```diff
@@ -1,35 +1,69 @@
-Write-Host ""
```
Comment (LittleLittleCloud) [inline scripts/prepdocs.ps1:1]: Do we need to update the corresponding .sh script as well?

Comment (jongio) [reply scripts/prepdocs.ps1:1]: No, this was just a pwsh issue.

**Review by luisquintanilla — APPROVED** [review]
Thanks @jongio. The deployment's working and @LittleLittleCloud #211 should fix the trigger issues.

**Pull request merged**